product_bundle/i18n/es.po

```diff
@@ -0,0 +1,208 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# * product_bundle
+# 
+# Translators:
+# Albert Cervera i Areny <albert@nan-tic.com>, 2009
+msgid ""
+msgstr ""
+"Project-Id-Version: odoo-addons (8.0)\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-09-09 23:57+0000\n"
+"PO-Revision-Date: 2015-09-10 00:00+0000\n"
+"Last-Translator: Juan Jose Scarafia <scarafia.juanjose@gmail.com>\n"
+"Language-Team: Spanish (http://www.transifex.com/adhoc/ingadhoc-odoo-addons-8-0/language/es/)\n"
```

Comment:
I believe that this module came, in some way, from product_pack (https://github.com/ingadhoc/product/tree/9.0/product_pack). What about authors?
Thanks!

Reply:
No, that's only something not updated. We created from the scratch, but @Yajo reuse the folder and didn't remove these files. You only need to check code to see that nothing is equal...

Reply:
ok, just wondering, no need to check code. Thanks!